src/installer.js

```diff
@@ -54,8 +54,13 @@ class DebianInstaller extends common.ElectronInstaller {
   /**
    * Copy the application into the package.
    */
-  copyApplication () {
-    return super.copyApplication(src => src !== path.join(this.options.src, 'LICENSE'))
+  async copyApplication () {
+    await super.copyApplication(src => src !== path.join(this.options.src, 'LICENSE'))
+    
+    // The sandbox helper executable must have the setuid (+s / 0o4000) bit set.
+    const sandboxHelperPath = path.join(this.stagingDir, 'chrome-sandbox')
+    if (await fs.exists(sandboxHelperPath))
+      await fs.chmod(sandboxHelperPath, 0o4755)
```

Comment:
I wonder if this should be extracted into a method in `electron-installer-common` so other modules like `electron-installer-redhat` and possibly `electron-installer-snap` / `electron-installer-flatpak` can use them.

Reply:
☝️ This is a Good Idea

Reply:
Should this be a separate method like `updateSandboxHelperPermissions` or a built-in part of `copyApplication`?

Reply:
Separate method. `electron-installer-common` is also used by `electron-installer-windows`.